sensei-lms.php

```diff
@@ -88,7 +88,7 @@ function is_sensei_activating( $activating_plugin, $plugin ) {
 	 * @since 1.8.0
 	 */
 	function Sensei() {
-		return Sensei_Main::instance( array( 'version' => '3.6.0-beta.1' ) );
+		return Sensei_Main::instance( array( 'version' => '3.7.0-dev' ) );
```

Comment:
Since this is needed here, we should bump the version everywhere in this PR to keep it consistent.

Reply:
Bumped here f70793c07cb36bee03418dd85e9719a73f8564a0